Shared/Realm.Sync.Shared/Handles/SharedRealmHandleExtensions.cs

```diff
@@ -59,6 +62,15 @@ public static SharedRealmHandle OpenWithSync(Realms.Native.Configuration configu
             return handle;
         }
 
+        public static string GetRealmPath(string userIdentity, Uri serverUri)
```

Comment:
It would be faster to pass in a `UserHandle` and get the identity from that instead of copying the string.